opencti-worker/src/worker.py

```diff
@@ -334,13 +334,10 @@ def data_handler(  # pylint: disable=too-many-statements, too-many-locals
         except Exception as ex:  # pylint: disable=broad-except
             error = str(ex)
             error_msg = traceback.format_exc()
-            if (
-                "LOCK_ERROR" in error_msg
-                and self.processing_count < MAX_PROCESSING_COUNT
```

Comment:
We remove this limit so the retry is indeed infinite. This is a valuable change even for regular lock error (happens when an entity is locked by another operation in progress).